installation/release-notes-main/release-notes-9-0-1.adoc

```diff
@@ -17,7 +17,7 @@ The release notes contain information to ensure the successful installation and
 
 == Description
 
-{PRODUCT_LONG}, {VERSION901}  fix release, provides important bug fixes requested by customers and the security fixes to PSIRTs/CVEs.
+{PRODUCT_LONG}, {VERSION901}  fix release, provides important bug fixes requested by customers and the security fixes to CVEs.
```

Comment:
This should be VERSION_901, otherwise it will result in a warning during the build.